### PR TITLE
Configure transform-runtime for all builds

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
Here's the demo: https://github.com/graingert/ng-redux/commit/a6f12618634c7be942935c862428a139ab63b03f#commitcomment-24111563

File: .babelrc [hunk 1]
```diff
@@ -7,14 +7,10 @@
       }
     ]
   ],
+  "plugins": "transform-runtime",
   "env": {
     "test": {
       "presets": ["env"]
-    },
-    "production": {
```
Comment (graingert) [inline .babelrc]: this "production" config was not being used.

File: rollup.config.js [hunk 2]
```diff
@@ -3,27 +3,24 @@ import commonjs from 'rollup-plugin-commonjs';
...
-  config.external = [
-    'invariant',
-    'lodash.curry',
-    'lodash.isfunction',
-    'lodash.isobject',
-    'lodash.isplainobject',
-    'lodash.map',
-    'redux',
-  ];
+  config.external = id => RegExp(`^(${externals})(\/.*)?$`).test(id);
```
Comment (deini) [inline rollup.config.js]: Is there any reason this is not just `config.external = Object.keys(dependencies)`?

File: .babelrc [hunk 3]
```diff
@@ -7,14 +7,10 @@
       }
     ]
   ],
+  "plugins": "transform-runtime",
```
Comment (deini) [inline .babelrc]: Should this be an array?

Comment (graingert) [reply .babelrc]: @deini it can be an array or a string. Happy to change it to an array.

File: rollup.config.js (hunk 2 above)
Comment (graingert) [reply rollup.config.js]: because then it would not externalize `package/foo` https://rollupjs.org/#peer-dependencies

Comment (deini) [reply rollup.config.js]: Ahh gotcha, that makes sense.

**Comment by graingert** [comment]
@deini any chance of another release? It's also probably worth deprecating 3.5.0 and 3.5.1

**Comment by deini** [comment]
Need to wait for @AntJanus

**Pull request merged**